src/main/java/com/cburch/logisim/vhdl/sim/VhdlSimulatorTclBinder.java

```diff
@@ -70,8 +70,7 @@ private void init(int serverPort) {
     builder = new ProcessBuilder(command);
 
     Map<String, String> env = builder.environment();
```

Comment:
@BFH-ktt1: Your fix is incomplete: `env` is not used elsewhere and can be removed, too.